deploy/yamls/node-disk-manager.yaml

```diff
@@ -34,6 +34,9 @@ spec:
         - -v=4
         - --feature-gates="GPTBasedUUID"
         - --feature-gates="APIService"
+        # Detect mount point and filesystem changes wihtout restart.
+        # Uncomment the line below to enable the feature.
+        # --feature-gates="MountChangeDetection"
```

Comment:
Need to run `make manifests` after adding this, So that the ndm-operator.yaml also gets updated.